recipe/meta.yaml

```diff
@@ -1,30 +1,32 @@
 {% set name = "dask-yarn" %}
-{% set version = "0.8.1" %}
+{% set version = "0.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 44ba7cd2e2ca6004f8cbcc358816fea68a898da509cd41dd85e7c4a7c7fcb4ed
+  sha256: e533be8ae0b1d46c7e916e50506a2b039de8d84a1590c705ad4166dda292623e
 
 build:
   number: 0
   skip: true  # [win or py<35]
   entry_points:
     - dask-yarn=dask_yarn.cli:main
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script:
+    - python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
+    - cp {{ SP_DIR }}/{{ name }}-{{ version }}.dist-info/LICENSE.txt {{ SRC_DIR }}/LICENSE.txt
```

Comment:
Think we can revert this if the `sdist` works. The license file is already included there and conda-build knows how to pick that up

Reply:
Looks like this reappeared, which causing the CI issue

Reply:
Fixed in latest commit